examples/core-api/checkout.php

```diff
@@ -1,17 +1,22 @@
 <?php
+// This is just for very basic implementation reference, in production, you should validate the incoming requests and implement your backend more securely.
+// Please refer to this docs:
+// https://docs.midtrans.com/en/core-api/credit-card?id=_1-getting-the-card-token
 
 namespace Midtrans;
 
 require_once dirname(__FILE__) . '/../../Midtrans.php';
-
-// YOUR CLIENT KEY
+// Set Your server key
 // can find in Merchant Portal -> Settings -> Access keys
-Config::$clientKey = "<your client key>";
-
+Config::$clientKey = '<your client key>';
 if (strpos(Config::$clientKey, 'your ') != false ) {
-    echo "<p style='background: #FFB588; padding: 10px;'>";
-    echo "Please set your client key in file " . __FILE__;
-    echo "</p>";
+    echo "<code>";
+    echo "<h4>Please set your server key from sandbox</h4>";
```

Comment:
message should be `client key`, not `server key`. Also refactor as `printExampleWarningMessage` function